plugins/pdr_deterministic_plugin/.pytest/run_pdr_standalone_pytest.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash -x
+
+PLUGIN_DIR="plugins/pdr_deterministic_plugin"
+pip install -r $PLUGIN_DIR/requirements.txt >/dev/null 2>&1
+
+cp -r utils $PLUGIN_DIR/ufm_sim_web_service
+cp -r utils $PLUGIN_DIR/tests
+
+echo "Init PDR configuration file"
+CONFIG_FILE="/config/pdr_deterministic.conf"
+mkdir -p /config
+cp -f $PLUGIN_DIR/build/config/pdr_deterministic.conf "$CONFIG_FILE"
+sed -i -e 's/\nTEST_MODE=True\n//g' "$CONFIG_FILE"
+echo -e '\n[Common]\nTEST_MODE=True\n' >> "$CONFIG_FILE"
```

Comment:
echoing without checking if it exists before, caused us trouble with running these tests multiple times previously.

Reply:
Line 
sed -i -e 's/\nTEST_MODE=\n//g' "$CONFIG_FILE"
will remove any existing TEST_MODE lines from the file